src/intra_edge.rs

```diff
@@ -234,7 +230,7 @@ impl DefaultValue for EdgeTip {
 }
 
 impl DefaultValue for EdgeBranch {
-    const DEFAULT: Self = Self::new(EdgeFlags::empty(), 0 as BlockLevel);
+    const DEFAULT: Self = Self::new(EdgeFlags::empty(), BlockLevel::DEFAULT);
```

Comment:
```suggestion
    const DEFAULT: Self = Self::new(EdgeFlags::empty(), BlockLevel::BL_128X128);
```

128 isn't an obvious default so I don't think we want to `impl DefaultValue` for it and we can just use the variant directly here.